modules/exchanging-identity-token.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-short-lived-access.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="exchanging-identity-token_{context}"]
+= Exchanging an identity token
+
+[role="_abstract"]
+
+.Prerequisites
+. You have to have a valid OpenID Connect (OIDC) token.
+. A *Machine access* configuration exists for the {product-title-short} instance you want to access.
+
+.Procedure
+. Prepare the POST request's JSON data as in the following example:
```

Comment:
Somewhat of a nit but if the JSON below was an example I'd expect it to have an actual example value rather than a user-replaced value like `<id_token>`. I think it's a good idea to keep the user-replaced value, in which case I think this step should be reworded to not use "example":
```suggestion
. Prepare the POST request's JSON data:
```